packages/trader/package.json

```diff
@@ -105,6 +106,9 @@
     "promise-polyfill": "^8.1.3",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
I thought we were going to hoist these dependencies?

Reply:
We are. the actual package would be resolved from root (../../../node_modules instead of ./node_modules). but the reference should be here.